src/arch/src/aarch64/regs.rs

```diff
@@ -6,7 +6,7 @@
 // found in the THIRD-PARTY file.
```

Comment:
oh, can also mention in the commit message that we're adding a new feature (I'm not a fan of no-verb sentences).